diff --color -ura phd2-2.6.13dev7/contributions/MPI_IS_gaussian_process/tests/gaussian_process/gp_guider_test.cpp phd2-2.6.13dev7.new/contributions/MPI_IS_gaussian_process/tests/gaussian_process/gp_guider_test.cpp
--- phd2-2.6.13dev7/contributions/MPI_IS_gaussian_process/tests/gaussian_process/gp_guider_test.cpp	2025-01-12 04:23:56.000000000 +0100
+++ phd2-2.6.13dev7.new/contributions/MPI_IS_gaussian_process/tests/gaussian_process/gp_guider_test.cpp	2025-09-10 17:45:32.907961693 +0200
@@ -37,6 +37,7 @@
 
 #include <gtest/gtest.h>
 #include <vector>
+#include <iomanip>
 #include <iostream>
 #include "gaussian_process_guider.h"
 
diff --color -ura phd2-2.6.13dev7/thirdparty/thirdparty.cmake phd2-2.6.13dev7.new/thirdparty/thirdparty.cmake
--- phd2-2.6.13dev7/thirdparty/thirdparty.cmake	2025-01-12 04:23:56.000000000 +0100
+++ phd2-2.6.13dev7.new/thirdparty/thirdparty.cmake	2025-09-10 17:47:01.501200848 +0200
@@ -489,7 +489,6 @@
     indi
     GIT_REPOSITORY https://github.com/indilib/indi.git
     GIT_TAG 856ac85b965177d23cd0c819a49fd50bdaeece60  # v2.0.5
-    GIT_SHALLOW 1
     CMAKE_ARGS -Wno-dev
       -DINDI_BUILD_SERVER=OFF
       -DINDI_BUILD_DRIVERS=OFF
@@ -601,7 +600,6 @@
     OGMAcamSDK
     GIT_REPOSITORY https://github.com/OGMAvision/OGMAcamSDK.git
     GIT_TAG 5e83d81384479b1684b97e9f6936e453050da4cb
-    GIT_SHALLOW 1
   )
   FetchContent_MakeAvailable(OGMAcamSDK)
   include_directories(${ogmacamsdk_SOURCE_DIR}/inc)
Only in phd2-2.6.13dev7.new/thirdparty: thirdparty.cmake~
