diff --color -ura indi_pylibcamera-2.6.4.orig/src/indi_pylibcamera/indi_pylibcamera.py indi_pylibcamera-2.6.4.new/src/indi_pylibcamera/indi_pylibcamera.py
--- indi_pylibcamera-2.6.4.orig/src/indi_pylibcamera/indi_pylibcamera.py	2024-05-15 12:03:34.000000000 +0000
+++ indi_pylibcamera-2.6.4.new/src/indi_pylibcamera/indi_pylibcamera.py	2024-09-21 15:36:41.137094538 +0000
@@ -533,7 +533,7 @@
     ps_ax = ps_ax.split("\n")
     pids_oldDriver = []
     for processInfo in ps_ax:
-        if ("python3" in processInfo) and (my_fileName in processInfo):
+        if ("python" in processInfo) and (my_fileName in processInfo):
             PID = int(processInfo.strip().split(" ", maxsplit=1)[0])
             if PID != my_PID:
                 logger.info(f'found old driver with PID {PID} ({processInfo})')
